tests/e2e/default/enketo/pregnancy-visit.wdio-spec.js

```diff
@@ -19,13 +19,25 @@ describe('Pregnancy Visit', () => {
     date_of_birth: moment().subtract(25, 'years').format('YYYY-MM-DD'),
     parent: { _id: healthCenter._id, parent: healthCenter.parent }
   });
+  const DOCS_TO_KEEP = [
+    ...Array.from(places.values()).map(doc => doc._id),
+    pregnantWoman._id,
+    'fixture:user:user1',
+    'org.couchdb.user:user1',
+    [/^form:/],
+  ];
```

Comment:
I sometimes wish we would have a pre-test snapshot of the docs and the after test would just revert to that snapshot. 
Not requesting a change here, just daydreaming.